easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.27.1'
+
+homepage = "https://github.com/arq5x/bedtools2"
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/releases/download/v%(version)s/']
+sources = ['bedtools-%(version)s.tar.gz']
+checksums = ['c8c2acbaf3f9cb92dcf8e5cd59af6b31ae9c4598efb786ba6c84f66ca72fafd9']
+
+buildopts = 'CXX="$CXX"'
+
+dependencies = [
+    ('BamTools', '2.5.1'),
```

Comment:
@fizwit The existing `BEDTools` easyconfig also has a dependency on `zlib`, please include it here too next to `BamTools`

Reply:
ok